spec/system/bulk_submission_index_spec.rb

```diff
@@ -130,5 +130,30 @@
         end
       end
     end
+
+    context "with an existing exhausted bulk_submission" do
+      before do
+        create(:bulk_submission, :with_original_file, :exhausted)
+      end
+
+      it "user can remove them" do
+        visit "/bulk_submissions"
+
+        within(".govuk-table") do
+          expect(page)
+            .to have_selector(".govuk-table__cell", text: Date.current.strftime("%d %b %Y"))
+            .and have_selector(".govuk-table__cell", text: "basic_bulk_submission.csv")
+            .and have_selector(".govuk-table__cell .govuk-tag.govuk-tag--red", text: /Exhausted/i)
+            .and have_selector(".govuk-table__cell", text: "Remove")
+
+          expect(page).not_to have_selector(".govuk-table__cell", text: "Cancel")
+          expect(page).not_to have_selector(".govuk-table__cell", text: "Download")
+
+          expect(page).to have_selector(".govuk-table__body tr")
+          click_button("Remove", match: :one)
+          expect(page).to have_no_selector(".govuk-table__body tr")
```

Comment:
**nit** clearly `have_no_selector` passes, but wondering what ,if any, difference there is to `not_to have_selector` and whether we should use one or other consistently

Reply:
👍 have amended to not_to have_selector